ingester-prometheus-metrics/README.md

```diff
@@ -0,0 +1,52 @@
+# Export Metrics
+
+Metrics monitoring is a critical component for ensuring optimal ingester performance and reliability. It provides:
+
+- Real-time visibility into ingester health and performance
+- Proactive monitoring capabilities for maintaining system stability
+- Rapid troubleshooting and issue resolution
+
+For detailed information about available metrics, including descriptions and usage guidelines, see our [comprehensive metrics documentation](../docs/metrics-display.md#list-of-metrics-constantly-updated).
+
+## Start Metrics Exporter
+
+To start an HTTP server that serves Prometheus metrics, initialize the metrics exporter before starting the GreptimeDB ingester:
+
+```java
+    MetricsExporter metricsExporter = new MetricsExporter(MetricsUtil.metricRegistry());
+    metricsExporter.init(ExporterOptions.newDefault());
+    // Start GreptimeDB ingester
+    // ...
+```
+
+## Get Metrics
+
+You can check the output of `curl http://<host>:<port>/metrics` by getting the latest metrics of the ingester.
+
+## Export Metrics to Prometheus
```

Comment:
How about exporting metrics to (maybe another) GreptimeDB, via Prometheus remote write?

Reply:
Pull mode is required; currently, GreptimeDB only supports Push mode.

Reply:
I can use Pushgateway to address this, maybe in the next PR.